Modix/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -147,11 +147,11 @@ public static IServiceCollection AddModix(this IServiceCollection services)
             return services;
         }
 
-        public static IServiceCollection AddStatsD(this IServiceCollection services)
+        public static IServiceCollection AddStatsD(this IServiceCollection services, IHostingEnvironment environment)
         {
             var cfg = new StatsdConfig { Prefix = "modix" };
 
-            if (string.IsNullOrWhiteSpace(cfg.StatsdServerName))
+            if (environment.IsDevelopment() && string.IsNullOrWhiteSpace(cfg.StatsdServerName))
```

Comment:
Shouldn't this block be inverted on environment.IsProduction() ?
I'm also curious why the need to check the environment, when you can just not set the statsd server name